pom.xml

```diff
@@ -45,15 +45,15 @@
 
         <!-- These are typically overridden with BOMs -->
         <vaadin.flow.version>6.0-SNAPSHOT</vaadin.flow.version>
-        <spring-boot.version>2.2.0.RELEASE</spring-boot.version>
+        <spring-boot.version>2.4.0.RELEASE</spring-boot.version>
```

Comment:
Shouldn't it be now `2.4.0` not `2.4.0.RELEASE` ?

Reply:
🤦 yes